packages/remediations/src/common/SystemsTable.js

```diff
@@ -0,0 +1,83 @@
+import React, { useEffect, useRef, useState } from 'react';
+import { Provider } from 'react-redux';
+import propTypes from 'prop-types';
+import promiseMiddleware from 'redux-promise-middleware';
+import ReducerRegistry from '@redhat-cloud-services/frontend-components-utilities/ReducerRegistry';
+import InventoryTable from '@redhat-cloud-services/frontend-components/Inventory/InventoryTable';
+import {
+    fetchSystemsInfo,
+    inventoryEntitiesReducer as entitiesReducer
+} from '../utils';
+
+const SystemsTable = ({ registry, allSystemsNamed, allSystems, hasCheckbox, disabledColumns, bulkSelect }) => {
+
+    const inventory = useRef(null);
+
+    const inventoryApi = useRef({});
+
+    const onRefresh = (options, callback) => {
+        if (!callback && inventory && inventory.current) {
+            inventory.current.onRefreshData(options);
+        } else if (callback) {
+            callback(options);
+        }
+    };
```

Comment:
Do you really need this?

Reply:
yeah, it seems to be not needed, thank you